stdlib/Distributed/test/distributed_exec.jl

```diff
@@ -1399,6 +1399,13 @@ let thrown = false
     @test thrown
 end
 
+# issue #34333
+let
+    @test fetch(remotecall(Float64, id_other, 1)) == Float64(1)
```

Comment:
This is fine but in the future it's better to use `===` here to check that the type also matches.